### PR TITLE
Add support for multiple geometries in 3D preview

### PR DESCRIPTION
**Pull request by stianjensen**
Use a dropdown to select between Box, Sphere, Cylinder, Torus and
TorusKnot.

### CODE REVIEW

**Comment by sigvef** [comment]
needs rebase

**Comment by stianjensen** [comment]
ping

File: src/THREEPreviewNode.js [hunk 1]
```diff
@@ -112,7 +122,31 @@
       material.alphaMap = updateMapBundle(
           this.alphaMap, this.getInput('alphamap'), repeat);
 
-      this.cube.material = material;
+      this.model.material = material;
+
+      var geometryName = this.getInput('geometry');
+      console.log('geo', geometryName);
```
Comment (sigvef) [inline src/THREEPreviewNode.js]: log

**Comment by stianjensen** [comment]
ping

**Pull request merged**